models/labels/sandwich_attackers/ethereum/labels_sandwich_attackers_ethereum.sql

```diff
@@ -0,0 +1,32 @@
+{{config(alias='sandwich_attackers_ethereum')}}
```

Comment:
minor style comment, just for readability, I'd add a bit more whitespace in the config blocks

```
{{ config(
      alias=...
  )
}}
```